packages/gatsby/src/internal-plugins/query-runner/page-query-runner.js

```diff
@@ -116,10 +112,7 @@ const findIdsWithoutDataDependencies = () => {
   // Get list of paths not already tracked and run the queries for these
   // paths.
   const notTrackedIds = _.difference(
-    [
-      ...Array.from(state.pages.values(), p => p.path),
-      ...[...state.staticQueryComponents.values()].map(c => c.jsonName),
-    ],
+    [...[...state.staticQueryComponents.values()].map(c => c.jsonName)],
```

Comment:
do we want to put this on multiple lines into some temporary variables? It might be more readable.